client/src/app/site/pages/meetings/pages/participants/pages/participant-list/components/participant-list/participant-list.component.ts

```diff
@@ -220,6 +224,26 @@ export class ParticipantListComponent extends BaseMeetingListViewComponent<ViewU
         }
     }
 
+    public isLockedOutToggleDisabled(user: ViewUser): boolean {
+        if (this.isMultiSelect) {
+            return true;
+        } else if (!this.filterForLockedOut(user)) {
+            return true;
+        } else {
+            return !this.operator.hasPerms(Permission.userCanManage);
+        }
+    }
+
+    private filterForLockedOut(user: ViewUser): boolean {
+        return (
+            this.operator.operatorId !== user.id &&
+            !user.organization_management_level &&
+            !user.group_ids().includes(this.activeMeeting.admin_group_id) &&
```

Comment:
With your change to `hasPermission` this should not be necessary anymore. The check in line 243 should be sufficient.

Reply:
I updated this, use group.hasPermission and excluded the explicit admin group check, it is in the hasPermission check.